spec/features/application_record_save_for_later_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
```

Comment:
In general, we should be consistent with `eq` and `eql`. I've been using `eql` - I'm open to arguments for using one or the other, but I think not both would be good.